.travis.yml

```diff
@@ -85,80 +88,4 @@ script:
-      mkdir Pencil2D.app/Contents/MacOS/plugins;
-      wget -O ffmpeg.7z https://evermeet.cx/ffmpeg/getrelease/7z;
-      wget -O ffmpeg.7z.sig https://evermeet.cx/ffmpeg/getrelease/7z/sig;
-      echo "trusted-key 0x476C4B611A660874" > ~/.gnupg/gpg.conf;
-      curl https://evermeet.cx/ffmpeg/0x1A660874.asc | gpg --import;
-      gpg --verify ffmpeg.7z.sig ffmpeg.7z;
-      7z x ffmpeg.7z -o"Pencil2D.app/Contents/MacOS/plugins";
-      rm ffmpeg.7z ffmpeg.7z.sig;
-
-      echo "Copying necessary Qt frameworks";
-      macdeployqt Pencil2D.app;
-      echo "applying macdeployqt fix";
-      wget https://github.com/aurelien-rainone/macdeployqtfix/archive/master.zip; unzip -x master.zip; python $TRAVIS_BUILD_DIR/build/Pencil2D/macdeployqtfix-master/macdeployqtfix.py $TRAVIS_BUILD_DIR/build/Pencil2D/Pencil2D.app/Contents/MacOS/Pencil2D /usr/local/Cellar/qt/5.9.1/;
-      echo "Removing files";
-      rm -rf macdeployqtfix-master;
-      rm master.zip;
-      cd ..;
-      echo "Zipping...";
-      zip -r "pencil2d-mac-$(date +"%Y-%m-%d").zip" Pencil2D/;
-    fi'
-  - echo "zipping done"
-  - ls
-
-  # Upload to google drive
-  - 'if [ \( "$TRAVIS_BRANCH" == "master" -o "$TRAVIS_BRANCH" == "release" \) -o "$FORCE_NIGHTLY_UPLOAD" == "yes" ]; then
-    echo "Initiate deployment on Google Drive";
-    cd "$TRAVIS_BUILD_DIR/util";
-    if [ "$TRAVIS_OS_NAME" == "linux" ]; then
-      python3 nightly-build-upload.py "$LINUX_NIGHTLY_PARENT" "$TRAVIS_BUILD_DIR/build/pencil2d-linux-$(date +"%Y-%m-%d").AppImage";
-    fi;
-    if [ "$TRAVIS_OS_NAME" == "osx" ]; then
-      python3 nightly-build-upload.py "$OSX_NIGHTLY_PARENT" "$TRAVIS_BUILD_DIR/build/pencil2d-mac-$(date +"%Y-%m-%d").zip";
-    fi;
-    echo "Operation done";
-  fi'
-  # Upload internal pull request to google drive
-  - 'if [ "$TRAVIS_PULL_REQUEST_SLUG" == "pencil2d/pencil" ]; then
-    echo "Initiate deployment on Google Drive";
-    cd "$TRAVIS_BUILD_DIR/util";
-    if [ "$TRAVIS_OS_NAME" == "linux" ]; then
-      mv "$TRAVIS_BUILD_DIR/build/pencil2d-linux-$(date +"%Y-%m-%d").AppImage" "$TRAVIS_BUILD_DIR/build/pencil2d-linux-pr-$TRAVIS_PULL_REQUEST.AppImage";
-      python3 pr-build-upload.py "$LINUX_PR_PARENT" "$TRAVIS_BUILD_DIR/build/pencil2d-linux-pr-$TRAVIS_PULL_REQUEST.AppImage";
-    fi;
-    if [ "$TRAVIS_OS_NAME" == "osx" ]; then
-      mv "$TRAVIS_BUILD_DIR/build/pencil2d-mac-$(date +"%Y-%m-%d").zip" "$TRAVIS_BUILD_DIR/build/pencil2d-mac-pr-$TRAVIS_PULL_REQUEST.zip";
-      python3 pr-build-upload.py "$OSX_PR_PARENT" "$TRAVIS_BUILD_DIR/build/pencil2d-mac-pr-$TRAVIS_PULL_REQUEST.zip";
-    fi;
-  fi'
-
-  # Deploy doxygen documentation to github pages
-  - 'if [ "$TRAVIS_BRANCH" == "master" -a "$TRAVIS_OS_NAME" == "linux" ]; then
-      cd "$TRAVIS_BUILD_DIR/util/docs";
-      for i in core svg xmlpatterns; do
-        curl -fsSLO "https://doc.qt.io/qt-5/qt$i.tags";
-      done;
-      ./documentation-deploy.sh;
-    fi'
+  - '"${TRAVIS_BUILD_DIR}/util/after-build.sh"'
```

Comment:
The outer quotes are YAML quotes, the inner ones are bash quotes. I think it’s normally a good habit to quote variables in bash by default, but in this case it’s indeed not strictly necessary.

Reply:
Yes quoting bash variables is a good habit, I did not know what those extra quotes were for though. This change was so it would match the other script calls in the file, but perhaps we should be quoting those too.